src/orbit-client/src/main/kotlin/orbit/client/execution/ExecutionHandle.kt

```diff
@@ -132,14 +132,18 @@ internal class ExecutionHandle(
                 //  logger.warn { "Methods tagged with @OnDeactivate may only take 0 or 1 argument(s)" }
 
 
-                if (isSuspended) {
-                    DeferredWrappers.wrapSuspend(
-                        it.method,
-                        instance,
-                        reasonArgs
-                    )
-                } else {
-                    DeferredWrappers.wrapCall(it.method.invoke(instance, *reasonArgs)).await()
+                try {
+                    if (isSuspended) {
+                        DeferredWrappers.wrapSuspend(
+                            it.method,
+                            instance,
+                            reasonArgs
+                        )
+                    } else {
+                        DeferredWrappers.wrapCall(it.method.invoke(instance, *reasonArgs)).await()
+                    }
+                } catch (ite: InvocationTargetException) {
+                    logger.info { "Exception caught on actor deactivation" }
```

Comment:
This should be a warning I think. And should write out the exception contents.

Reply:
I can do that...